### PR TITLE
fix: add Abhishek Kushwaha

### PR DESCRIPTION
**Pull request by Abbhiishek**
## Changes proposed
- Added my own developer profile on LinkFree

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/790"></a>

### CODE REVIEW

File: public/data/Abhishek-Kushwaha.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/abbhishek_k",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Data Structure & Algorithms",
```
Comment (Panquesito7) [inline public/data/Abhishek-Kushwaha.json:20]:
```suggestion
      "title": "Started Data Structures & Algorithms",
```

File: public/data/Abhishek-Kushwaha.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      "date": "November  2021",
```
Comment (Panquesito7) [inline public/data/Abhishek-Kushwaha.json:21]:
```suggestion
      "date": "November 2021",
```

File: public/data/Abhishek-Kushwaha.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "icon": "YouTube",
+      "color": "red",
+      "description": "Started ",
```
Comment (Panquesito7) [inline public/data/Abhishek-Kushwaha.json:24]: Please provide more description about this.

File: public/data/Abhishek-Kushwaha.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    {
+      "title": "Started Open Source",
+      "date": "December 2021",
+      "icon": "GitHub",
+      "color": "green",
+      "description": "Started ",
```
Comment (Panquesito7) [inline public/data/Abhishek-Kushwaha.json:31]: Please provide more description about this.

File: public/data/Abhishek-Kushwaha.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    {
+      "title": "Learning Api Development",
+      "date": "December 2021",
+      "icon": "Globe",
+      "color": "orange",
+      "description": "Learning ",
```
Comment (Panquesito7) [inline public/data/Abhishek-Kushwaha.json:38]: Please provide more description about this.

File: public/data/Abhishek-Kushwaha.json [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    {
+      "title": "Started Discord Bot Development",
+      "date": "August 2021",
+      "icon": "Discord",
+      "color": "green",
+      "description": "Still Learning ",
```
Comment (Panquesito7) [inline public/data/Abhishek-Kushwaha.json:45]: Please provide more description about this.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Abbhiishek** [comment]
Done with requested Changes 😁

**Review by shreyaansjain06 — APPROVED** [review]
LGTM

**Review by krishguptadev — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
LGTM!

**Comment by Abbhiishek** [comment]
> LGTM!

I didn't get it ?
LGTM means?

**Comment by Vyvy-vi** [comment]
Looks Good To Me

**Comment by Vyvy-vi** [comment]
Could you resolve the linting errors? -
```
public/data/Abbhiishek.json
Error:   24:49  error  Trailing comma  json/*
Error:   31:72  error  Trailing comma  json/*
Error:   38:46  error  Trailing comma  json/*
Error:   45:44  error  Trailing comma  json/*
```

**Review by krishguptadev — APPROVED** [review]
Let's just wait for the workflows to run

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**